base/src/main/java/com/smartdevicelink/managers/screen/BaseTextAndGraphicManager.java

```diff
@@ -444,12 +472,30 @@ SdlArtwork getPrimaryGraphic() {
 	void setSecondaryGraphic(SdlArtwork secondaryGraphic) {
 		this.secondaryGraphic = secondaryGraphic;
 		if (!batchingUpdates) {
-			sdlUpdate(null);
+			sdlUpdate(true, null);
 		} else {
 			isDirty = true;
 		}
 	}
 
+	void changeLayout(TemplateConfiguration templateConfiguration, CompletionListener listener) {
```

Comment:
In my testing on Core 6+, I noticed that, if I change only colors and keep the template the same, the update fails with success = false.